go/vt/vttablet/tabletmanager/state_change.go

```diff
@@ -220,7 +220,7 @@ func (agent *ActionAgent) changeCallback(ctx context.Context, oldTablet, newTabl
 			}
 			srvKeyspace, err := agent.TopoServer.GetSrvKeyspace(ctx, newTablet.Alias.Cell, newTablet.Keyspace)
 			if err != nil {
-				log.Error("Fail to get SrvKeyspace")
+				log.Errorf("Fail to get SrvKeyspace %v", err)
```

Comment:
Nit: seems like "Fail to get SrvKeyspace: %v" with ':' might be better than "Fail to get SrvKeyspace %v"

Reply:
Also: bonus points for changing this to "Failed" instead of "Fail"